src/Apps/Order/Routes/Shipping2/index.tsx

```diff
@@ -552,22 +529,11 @@ export const ShippingRoute: FC<ShippingProps> = props => {
     }
   }, [currentStep, fulfillmentFormHelpers.isValid, selectedShippingQuoteId])
 
-  const orderContext: ShippingContextProps = useMemo(
-    () =>
-      pick(
-        savedOrderData,
-        "lockShippingCountryTo",
-        "isArtsyShipping",
-        "availableShippingCountries",
-        "shipsFrom",
-        "requiresArtsyShippingTo",
-        "selectedSavedAddressId",
-        "fulfillmentDetails",
-        "fulfillmentType"
-      ),
-    [savedOrderData]
-  )
-
+  const maybeShippingQuotesConnectionEdges =
+    order.lineItems?.edges?.[0]?.node?.shippingQuoteOptions?.edges
+  const shippingQuotesConnectionEdges = maybeShippingQuotesConnectionEdges
+    ? compact(maybeShippingQuotesConnectionEdges)
+    : ([] as NonNullable<typeof maybeShippingQuotesConnectionEdges>)
```

Comment:
a good amount of deletions in this file due to simplifying the context, but we still manually pull this stuff apart because the recipient of this prop on L534 is shared with the old shipping route.